### ISSUE TITLE
Unknown error type: Wait timeout (nvim_ui_attach)

### ISSUE THREAD

**Issue by passboltUser** [daa84/neovim-gtk#146]
I'm getting an error now. It worked fine just a few days ago. Error:

> Unknown error type: Wait timeout (nvim_ui_attach)

![error](https://user-images.githubusercontent.com/29798595/45537048-86c0e000-b803-11e8-8ca2-0d0ee7fce730.png)

I created a folder under `C:\Neovim`, extracted [neovim](https://github.com/neovim/neovim/releases/download/v0.3.1/nvim-win64.zip) there, added `C:\Neovim\bin` to the systemvariables PATH (calling `nvim` from anywhere within the powershell works, so I know `nvim` should also be available for `neovim-gtk` available).

**Comment by Anrock** [comment]
Same here. Though for me this started happening some way earlier. After couple of restarts it usually starts working.

**Comment by daa84** [comment]
This error means that nvim is found but connection took to long. In some cases this can be because of error in configuration.
Another reason can be that start takes to long, you can try to extend wait time with `--timeout=10`, 10 seconds is default value.

**Comment by passboltUser** [comment]
@daa84 Yes, the error is different and was also guessing that `nvim` can be found but for some reason can't be loaded. I set `--timeout=100` and I downloaded both packages so they are fresh and don't have any configuration issues. But I get unfortunately still the same error.
@Anrock I did like 5 restarts after I noticed the mentioned error, unfortunately this error is still there.

**Comment by passboltUser** [comment]
I also remember I opened the Plugins Manager, and downloaded `airline`, maybe that broke it. But since I freshly downloaded both nvim and nvim-gtk archives and I'm getting still the same error, `airline` may be stored somewhere else?

**Comment by daa84** [comment]
Try to disable Plugin Manager.
Try to remove files from `~/.config/nvim-gtk` (don't remove them completely just move it to some place as you loose your settings)
Looks like some configuration problem. But as neovim currently does not support pass start error messages to gui it is hard to find problem source. In case problem in init.vim - start of nvim from console will show errors

**Comment by passboltUser** [comment]
Ok, now we know more: Thanks, I didn't know it was under `$env:userprofile/.config/nvim-gtk`. I emptied it, started `nvim-gtk` again and it works. The same error comes again if I open the Plugin Manager and click the slider on the top, so it shows a blue "ON" (so the error appears without clicking any `Install` button).

**Comment by daa84** [comment]
For plugin manager to work it is need to install plug.vim to neovim first. Currently plug.vim preinstalled only in flatpak package

**Comment by passboltUser** [comment]
I downloaded [plug.vim](https://github.com/junegunn/vim-plug) and put it, as described, into the `autoload` folder (`%AppData%\..\Local\nvim\autoload`). When I enable the Plugin Manager by clicking on the slider at the top and set it to `ON`, I get this error:
![plug](https://user-images.githubusercontent.com/29798595/45611406-a48a8100-ba5f-11e8-92c1-bdb0b3c75dbd.png)

**Comment by passboltUser** [comment]
I installed [git](https://git-scm.com/download/win) and don't get the red error now. Unfortunately I still can't install `vim-airline` when I click the Install button and unfortunately I don't get any error messages this time so I don't know what's wrong.

**Comment by daa84** [comment]
Install button does not really install, but generate config, you still need to run plug commands to manage plugins (like PlugInstall, PlugUpdate). So currently this is only config generator not more.
